modules/cluster-logging-upgrading-logging.adoc

```diff
@@ -5,98 +5,135 @@
+* Back up your Elasticsearch and Kibana data.
 
 .Procedure
 
 . Update the Elasticsearch Operator:
 
-.. From the web console, click *Operators* -> *Installed Operators*.
+.. From the OpenShift web console, click *Operators* -> *Installed Operators*.
 
-.. Select the `openshift-logging` project.
+.. Select the `openshift-operators-redhat` project.
 
-.. Click the *Elasticsearch Operator*.
+.. If the Elasticsearch Operator is installed on the `openshift-logging` namespace, uninstall the Elasticsearch Operator.
 
-.. Click *Subscription* -> *Channel*.
+[NOTE]
+====
+This uninstall action does _not_ delete the existing data, volumes, and so on.
+====
 
-.. In the *Change Subscription Update Channel* window, select *4.3* and click *Save*.
+.. Click *Operators* -> *OperatorHub*. Search for and select `OpenShift Elasticsearch Operator`.
+
+ .. Click *Install*
+ +
+ The *Install operator* page opens.
+
+.. For *Update Channel*, make sure *4.6* is selected.
+
+.. For *Installed Namepsace*, select `openshift-operators-redhat` namespace.
+
+.. Click *Install*.
 
 .. Wait for a few seconds, then click *Operators* -> *Installed Operators*.
 +
-The Elasticsearch Operator is shown as 4.3. For example:
+The Elasticsearch Operator is shown as 4.6. For example:
 +
+[source,terminal]
 ----
 Elasticsearch Operator
-4.3.0-201909201915 provided
+4.6.0-202007012112.p0 provided
 by Red Hat, Inc
 ----
++
+Wait for the *Status* field to report *Succeeded*.
 
 . Update the Cluster Logging Operator:
 
-.. From the web console, click *Operators* -> *Installed Operators*.
+.. From the OpenShift web console, click *Operators* -> *Installed Operators*.
 
-.. Select the `openshift-logging` Project.
+.. Select the `openshift-logging` project.
 
-.. Click the *Cluster Logging Operator*.
+.. Click *Cluster Logging*.
 
 .. Click *Subscription* -> *Channel*.
 
-.. In the *Change Subscription Update Channel* window, select *4.3* and click *Save*.
+.. In the *Change Subscription Update Channel* window, select *4.6*.
+
+.. On the *YAML* tab, set the retention policy in the `clusterlogging` custom resource (CR) as follows. You must specify the length of time that Elasticsearch should retain each log source. Enter an integer for the number of days, for example, `7d` for seven days. The `application` `maxAge` can be less than or equal to `7d`. The `infra` `maxAge` and `audit` `maxAge` must be `0d`, and you must *explicitly set this value* to override the default. Logs older than the `maxAge` are deleted. You must specify a retention policy for each log source or the Elasticsearch indices will not be created for that source.
++
+[source,terminal]
+----
+kind: "ClusterLogging"
+metadata:
+  name: "instance"
+  namespace: "openshift-logging"
+spec:
+  managementState: "Managed"
+  logStore:
+    type: "elasticsearch"
+    retentionPolicy:
+      application:
+        maxAge: 7d
+      infra:
```

Comment:
@vikram-redhat  Per this Bugzilla: https://github.com/openshift/openshift-docs/issues/30300

please change these values to:

infra:
        maxAge: 1h
      audit:
        maxAge: 1h
        
  I will make these same changes in the other topic that this BZ impacts (OSD Install CLuster Logging).